core/src/com/po/fuck/model/weapons/CosmicBullet.java

```diff
@@ -13,8 +14,9 @@ public final class CosmicBullet extends Bullet {
         damage = COSMIC_BULLET_DAMAGE;
         lifeTime = COSMIC_BULLET_LIFE_TIME;
         geometryData = new GeometryData();
-        geometryData.setWidth(SpriteInfo.getBasicAssetInfo(this.getClass()).getWidth());
-        geometryData.setHeight(SpriteInfo.getBasicAssetInfo(this.getClass()).getHeight());
+        BasicSpriteInfo thisInfo = SpriteInfo.getBasicAssetInfo(this.getClass());
```

Comment:
Move `getBasicAssetInfo` to another class. SpriteInfo is responsible only for storing info.